robosuite/scripts/demo_device_ik_control.py

```diff
@@ -89,7 +101,9 @@
 
         # rotate the gripper so we can see it easily
         if env.mujoco_robot.name == 'sawyer':
-            env.set_robot_joint_positions([0, -1.18, 0.00, 2.18, 0.00, 0.57, 1.5708])
+            # TODO: Confirm that this is no longer necessary
```

Comment:
can we follow up on this TODO?

Reply:
this file is no longer relevant; it's been refactored as `demo_device_control.py` since it also includes osc demo functionality

Reply:
Is the plan to remove this demo script and replace it with a new demo script for all controllers?

Reply:
Yes; this PR doesn't have `demo_device_ik_control.py` in its repo -- it only has `demo_device_control.py`

Reply:
I don't think it makes sense to have a demo device script for all controllers since I'm not sure how / why users would want to try to directly control joint torques / velocities / positions? Unless you had specific thoughts in mind

Reply:
Great, `demo_device_control.py` is what exactly I want for replacement of `demo_device_ik_control.py`